contrib/jmx-metrics/src/main/java/io/opentelemetry/contrib/jmxmetrics/JmxClient.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.contrib.jmxmetrics;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.security.Provider;
+import java.security.Security;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.annotation.Nullable;
+import javax.management.MBeanServerConnection;
+import javax.management.ObjectName;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorFactory;
+import javax.management.remote.JMXServiceURL;
+
+public class JmxClient {
+  private static final Logger logger = Logger.getLogger(JmxClient.class.getName());
+
+  private final JMXServiceURL url;
+  private final String username;
+  private final String password;
+  private final String realm;
+  private final String jmxRemoteProfiles;
+  @Nullable private JMXConnector jmxConn;
+
+  JmxClient(final JmxConfig config) throws MalformedURLException {
+    setSystemProperties(config);
+    this.url = new JMXServiceURL(config.serviceUrl);
+    this.username = config.username;
+    this.password = config.password;
+    this.realm = config.realm;
+    this.jmxRemoteProfiles = config.jmxRemoteProfiles;
+  }
+
+  private static void setSystemProperties(final JmxConfig config) {
+    if (!JmxConfig.isBlank(config.keyStorePath)) {
+      System.setProperty("javax.net.ssl.keyStore", config.keyStorePath);
```

Comment:
If we're only forwarding the properties anyways, I think we don't need to define our own property names, users should generally be more familiar with the standard props anyways. And we're not preventing collission if we copy them in so there doesn't seem to be value in the otel versions.

Reply:
This is much more straightforward, and I've adopted this request.